src/screens/OtherProfile/index.js

```diff
@@ -420,7 +422,8 @@ const OtherProfile = () => {
     };
     const result = await setUnFollow(data);
     if (result.code === 200) {
-      fetchOtherProfile();
+      handleUpdateDiscoveryUser(other_id, false);
+      fetchOtherProfile(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fetchOtherProfile` function is now called with `true` as an argument, which means caching will be prevented when unfollowing a user. This seems logical as you would want to get the most recent data after such an action. The `handleUpdateDiscoveryUser` function is also being called with new parameters. Ensure these changes align with your application logic.
```diff
- fetchOtherProfile();
+ handleUpdateDiscoveryUser(other_id, false);
+ fetchOtherProfile(true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->